schema/omega-services.async.api.yaml

```diff
@@ -777,6 +830,199 @@ components:
+        creator:
+          type: array
+          items:
+            $ref: '#/components/schemas/GeneralIdentifier'
+          description: the identifier of the Creating Body of the Record concept, i.e. a URI to an Agent
+        current-description:
+          $ref: '#/components/schemas/GeneralIdentifier'
+          description: the identifier of the current description of the Record concept
+        description:
+          type: array
+          items:
+            $ref: '#/components/schemas/RecordDescriptionSummary'
+
+    RecordFull:
+      type: object
+      required:
+        - identifier
+        - type
+        - creator
+        - current-description
+        - description
+      properties:
+        identifier:
+          $ref: '#/components/schemas/GeneralIdentifier'
+        type:
+          $ref: '#/components/schemas/RecordType'
+        creator:
+          type: array
+          items:
+            $ref: '#/components/schemas/GeneralIdentifier'
+          description: the identifier of the Creating Body of the Record concept, i.e. a URI to an Agent
+        current-description:
+          $ref: '#/components/schemas/GeneralIdentifier'
+          description: the identifier of the current description of the Record concept
+        description:
+          type: array
+          items:
+            $ref: '#/components/schemas/RecordDescriptionFull'
+
+    RecordDescriptionSummary:
+      type: object
+      required:
+        - identifier
+        - label
+        - abstract
+        - access-rights
+        - is-part-of
+        - version-timestamp
+      properties:
+        identifier:
+          $ref: '#/components/schemas/GeneralIdentifier'
+        secondary-identifier:
+          type: array
+          items:
+            $ref: '#/components/schemas/GeneralTypedIdentifier'
+        label:
+          $ref: '#/components/schemas/GeneralLabel'
+        abstract:
+          $ref: '#/components/schemas/GeneralDescription'
+        access-rights:
+          type: array
+          items:
+            $ref: '#/components/schemas/GeneralIdentifier'
+          description: the identifier of the Rights Statement for the Record description
+        is-part-of:
+          type: array
+          items:
+            $ref: '#/components/schemas/GeneralIdentifier'
+          description: Links to the parent of which this is part of
+        previous-sibling:
+          $ref: '#/components/schemas/GeneralIdentifier'
+          description: Links to the previous sibling. Used when there is an order of resources to be applied across siblings
+        version-timestamp:
+          $ref: '#/components/schemas/VersionTimestamp'
+        previous-description:
+          $ref: '#/components/schemas/GeneralIdentifier'
+
+    RecordDescriptionFull:
+      type: object
+      allOf:
+        - $ref: '#/components/schemas/RecordDescriptionSummary'
+        - $ref: '#/components/schemas/RecordDescriptionProperties'
+
+    RecordDescriptionProperties:
```

Comment:
Nothing in `RecordDescriptionProperties` is _required_. Is that correct?

Reply:
Yes, that is correct.